modules/openapi-generator/src/main/resources/JavaSpring/api.mustache

```diff
@@ -8,11 +8,11 @@ package {{package}};
 {{#imports}}import {{import}};
 {{/imports}}
 {{#oas3}}
+import io.swagger.v3.oas.annotations.Hidden;
 import io.swagger.v3.oas.annotations.Operation;
 import io.swagger.v3.oas.annotations.Parameter;
 import io.swagger.v3.oas.annotations.media.Content;
 import io.swagger.v3.oas.annotations.media.Schema;
-import io.swagger.v3.oas.annotations.parameters.RequestBody;
```

Comment:
I think there's value in having both annotations present - but open to removing if no one else feels the same way

Reply:
This was my proposed fix

https://github.com/OpenAPITools/openapi-generator/pull/10766/files

Reply:
I would address issue when we would actually need `io.swagger.v3.oas.annotations.parameters.RequestBody`. So far, the import is unused, hence the removal.

Reply:
It is used - by deleting this import the "@RequestBody" annotation is changing to

Reply:
And this is how it should be - `org.springframework.web.bind.annotation.RequestBody` ist the annotation required technically by Spring Mvc or OpenFeign to handle a request. `io.swagger.v3.oas.annotations.parameters.RequestBody` is never used, as far as I can see.

If we need them both, they should be fully qualified

```
    @org.springframework.web.bind.annotation.RequestBody   // Spring
    @io.swagger.v3.oas.annotations.parameters.RequestBody  // Swagger
```
Question is : Do we actually need to support `@io.swagger.v3.oas.annotations.parameters.RequestBody`? What is it good for? Do Tools like SpringFox or SpringDoc support the annotation?

Reply:
I believe it is supported by SpringDoc

Reply:
Thoughts on hvaing both annotations?